OpenRA.Mods.Common/Traits/Harvester.cs

```diff
@@ -279,7 +278,10 @@ void INotifyIdle.TickIdle(Actor self)
 				return;
 			}
 
-			UnblockRefinery(self);
+			// We don't need to do this every idle tick, unblocking on first idle tick should be sufficient
```

Comment:
I'm not sure I agree with this: the harvester may be boxed in and unable to move in the first tick, and then have a route open up later.